classes/QueryOutput.php

```diff
@@ -162,6 +168,13 @@ public function render_query( $query, $type = '' ) {
 				</div>
 			<?php endif; ?>
 
+			<?php if ( $context ) : ?>
+				<div class="ep-query-type">
```

Comment:
Can we change the class name to `ep-query-context`

Reply:
Good catch, @burhandodhy, thanks. I've updated it in [3e97580](https://github.com/10up/debug-bar-elasticpress/pull/82/commits/3e97580b3aed4f9be7e5779d3aafc5b2d0f069ef)